### ISSUE TITLE
[Bug] Chart Screen Distortion on Big Data Chart during DataZoom

### ISSUE THREAD

**Issue by bbeas** [apache/echarts#19524]
### Version

5.4.3

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?c=grid-multiple

### Steps to Reproduce

1. Quickly scroll to zoom in and zoom out the chart

### Current Behavior

Seems like the dots of the previous frame didn't completely disappear before the next frame is rendered.
![20240118-110753](https://github.com/apache/echarts/assets/29697116/4c2929e5-958e-4dd4-8e8f-8e0e060b8306)

### Expected Behavior

The line should be rendered only once.

### Environment

```markdown
- OS: MacOS 12.7.2
- Browser: Version 120.0.6099.129 (Official Build) (arm64)
- Framework: React
```

### Any additional comments?

This problem can be simply reproducted in the echarts official demo.

**Comment by helgasoft** [comment]
usually ```animation: false``` fixes similar problems
📌 please close issue if problem solved.

**Comment by plainheart** [comment]
This bug has been fixed in the coming v5.5.0, you can try the latest nightly 5.4.4 version to help check if it persists.
